src/Eppie.App/Eppie.App/Services/AppStoreService.cs

```diff
@@ -18,16 +18,43 @@
 
 using Tuvi.App.ViewModels.Services;
 using Windows.Services.Store;
+using Windows.System;
 
 namespace Tuvi.App.Shared.Services
 {
-    public class PurchaseService : IPurchaseService
+    public class AppStoreService : IAppStoreService
     {
         public Task BuySubscriptionAsync()
         {
             var subscription = new SubscriptionProduct("<InAppOfferToken>", "<ProductId>");
```

Comment:
Add readonly field for "<InAppOfferToken>" and "<ProductId>"

Reply:
Done